components/ws-daemon/pkg/content/initializer.go

```diff
@@ -398,7 +398,9 @@ func (rs *remoteContentStorage) Download(ctx context.Context, destination string
 	tempFile.Close()
 
 	args := []string{
-		"-x16", "-j12",
+		"-s10", "-x16", "-j12",
```

Comment:
@utam0k look here, `-s10` is also relevant

Reply:
I'm wondering if the non-abbreviated one (`--split`) would be easier to understand the code than the abbreviated one, can I make a PR?

Reply:
Honestly, even with the words, I have to read description to know what they do. :sweat: In other words, I'd still need the website, or help from the command, to know how to tune.

Reply:
Unfortunately, I agree with you :+1: